testfiles/TF06/06.2-all-fail-1.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Decorative Image - non-descriptive text in accessible name and description (TC06.2-all-fail-1)</title>
```

Comment:
this one is "with no accessible name or description defined, not in background", meaning an <img> with no attribute to define accname